### DISCUSSION TITLE
npm installation fails, because model-viewer depends on the old 157 version of Three JS

### DISCUSSION THREAD

**Discussion by teneon** [google/model-viewer#4602] [Q&A] [answered]
### Description
Hi there, i am on ubuntu 22.04 ,i have Three JS version 159  installed.

When try to add model-viewer (currently latest version 3.3.0) 
`npm i @google/model-viewer@3.3.0`

i get the following error:

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: myproject@0.0.1
npm ERR! Found: three@0.159.0
npm ERR! node_modules/three
npm ERR!   three@"^0.159.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer three@"^0.157.0" from @google/model-viewer@3.3.0
npm ERR! node_modules/@google/model-viewer
npm ERR!   @google/model-viewer@"3.3.0" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.

```

How can i fix this, i wouldn't want to downgrade Three JS to 157 version of course and i find it weird to require 157, because in package.json as dependency for model-viewer the 159 version is listed.

Any help appreciated! :-)

best regards,
Jim

**Comment by elalish** [comment, marked as answer]
You're probably looking at the latest master version of MV, not v3.3.0 - pretty sure that one is indeed on r157. You have several options - wait for our next release, which will bump the three.js version, or downgrade your own (generally not a big problem unless you need a very recent feature). I believe there are even ways to force npm to ignore the version discrepancy, though do so at your own risk - we only test with the version of three.js we depend on.
